package/MDAnalysis/analysis/__init__.py

```diff
@@ -27,6 +27,20 @@
 
 The :mod:`MDAnalysis.analysis` sub-package contains various recipes and
 algorithms that can be used to analyze MD trajectories.
+If not stated differently, an analysis conducted by the available modules
+always follows the same structure
+
+1. Initialize the object previously imported.
+2. Run the analysis for specific trajectory slices
+3. Access the analysis from the `results` attribute (if available)
+
+ .. code-block:: python
+
+    from MDAnalysis.anlaysis import AnalysisModule
+
+    analysis_obj = AnalysisModule(trajectory, ...)
+    analysis_obj.run(start_frame, stop_frame, step)
```

Comment:
```suggestion
    from MDAnalysis.analysis import ExampleAnalysisModule  # (e.g. RMSD)

    analysis_obj = ExampleAnalysisModule(universe, ...)
    analysis_obj.run(start_frame, stop_frame, step)
```

I think it's best to be clear. Usually analysis modules operate on universes, right? There's a `universe.trajectory` attribute that would be best not to mix up with it.